mysql-test/suite/galera/t/galera_sst_rsync.cnf

```diff
@@ -3,6 +3,9 @@
 [mysqld]
 wsrep_sst_method=rsync
 
+[sst]
```

Comment:
since we're not using the ssl_mode, should this just be "encrypt=0"

Reply:
You're right! Thanks for catching this. I'll update it soon.